Simplenote/Classes/SPNoteListViewController+Extensions.swift

```diff
@@ -636,12 +590,23 @@ private extension SPNoteListViewController {
     ///
     func prefixText(for note: Note) -> String? {
         guard case .searching = notesListController.state,
-            let date = note.date(for: notesListController.searchSortMode)
+            let date = note.date(for: notesListController.sortMode)
             else {
                 return nil
         }
 
-        return DateFormatter.listDateFormatter.string(from: date)
+        return dateSortPrefix() + DateFormatter.listDateFormatter.string(from: date)
```

Comment:
Let's preserve the original behavior, and address the Timestamp changes in a follow up PR.

Ref. here `p2XJRt-2RQ-p2`